adding scheduler cli-examples (#8983)
aws/aws-cli: awscli/examples/scheduler/delete-schedule-group.rst

**To delete the specified schedule group**

The following ``delete-schedule-group`` example deletes the specified schedule group. Deleting a schedule group deletes all schedules associated with the group in EventBridge. ::

    aws scheduler delete-schedule-group \
        --name DailyLambda

This command produces no output.

> For more information, see `Deleting a schedule group in EventBridge Scheduler <https://docs.aws.amazon.com/scheduler/laDailyLambda/UserGuide/managing-schedule-group-delete.html>`__ in the *Amazon EventBridge Scheduler User Guide*.

Reviewer: As of writing, the link https://docs.aws.amazon.com/scheduler/laDailyLambda/UserGuide/managing-schedule-group-delete.html returns a 404 Not Found

Author: updated

**To delete the specified schedule group**

> The following ``delete-schedule-group`` deletes the specified schedule group. Deleting a schedule group results in EventBridge Scheduler deleting all schedules associated with the group. If the command succeeds, no output is returned. ::

Reviewer 2: ```suggestion
The following ``delete-schedule-group`` example deletes the specified schedule group. Deleting a schedule group deletes all schedules associated with the group in EventBridge. ::
```

Author: file fixed

	aws scheduler delete-schedule-group \

> 		--name DailyLambda

Reviewer 2: ```suggestion
		--name DailyLambda

This command produces no output.
```
